widgets/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/validators/RunOnceEditorValidator.java

```diff
@@ -48,4 +48,12 @@ public boolean isValid() {
 
   public void clear() {
   }
+
+  private String normalizeDateEn (String dateString){
+    if(dateString != null && !dateString.isEmpty()){
+      return dateString.replace( "vorm." , "am").replace( "nachm." , "pm") ;
+    }else {
+      return dateString;
+    }
+  }
```

Comment:
Formatting is off.

Reply:
Also, this implementation looks to be, essentially, a hack. Could a locale-aware date function be used instead?